jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -120,12 +120,12 @@ def __init__(self, *args, **kwargs):
             sv.state.add_callback("x_max",
                                   lambda x_max: self._on_spectrum_viewer_limits_changed())
 
-        self._set_relevant()
+        self.observe_traitlets_for_relevancy(non_empty_traitlets=['viewer_items'],
```

Comment:
i see in most other plugins this is called after checking if the app is deconfigged, just curious why that's not the case here?

Reply:
the intention here was to keep the old behavior in configs (where most plugins do show at the start, even if the content is empty or just displays a message), while having full context-aware support in deconfigged.  There is a chance though that some of the if-statements could be removed with testing that it doesn't break workflows.